lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -284,28 +281,22 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
-  newproperty(:cap_add_paths_receive) do
-    desc 'cap_add_paths_receive state. ' \
-         "Valid values are true, false or 'default'"
-    newvalues(:true, :false, :default)
-  end
-
-  newproperty(:cap_add_paths_receive_disable) do
-    desc 'cap_add_paths_receive_disable state. ' \
-         "Valid values are true, false or 'default'"
-    newvalues(:true, :false, :default)
-  end
-
-  newproperty(:cap_add_paths_send) do
-    desc 'cap_add_paths_send state. ' \
-         "Valid values are true, false or 'default'"
-    newvalues(:true, :false, :default)
+  newproperty(:additional_paths_receive) do
+    desc 'additional_paths_receive state. ' \
+         "Valid values are 'enable' for basic command enablement; 'disable' " \
+         "for disabling the command at the neighbor_af level; and 'inherit' " \
+         'to remove the command at this level.'
```

Comment:
Hmm, I think the updates haven't been pushed as the PR diff is still showing `remove the command at this level`